benchmarks/fci_benchmarks.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2021 Satpy developers
+#
+# This file is part of satpy.
+#
+# satpy is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later
+# version.
+#
+# satpy is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# satpy.  If not, see <http://www.gnu.org/licenses/>.
+"""Benchmark FCI.
+
+Benchmarks for reading and processing data from the Meteosat Third Generation
+(MTG) Flexible Combined Imager (FCI).  Uses pre-launch simulated test data as
+published by EUMETSAT in 2020.
+
+Also includes some benchmarks trying different resamplers.
+"""
+
+import fnmatch
+import os
+
+import satpy
+import satpy.demo.fci
+
+
+class FCI:
+    """Benchmark FCI FDHSI test data reading."""
+
+    timeout = 600
+    data_files = []
+    region = "maspalomas"
+
+    def setup(self, *args):
+        """Fetch the data files."""
+        fns = self.get_filenames()
+        cnt = len(fns)
+        if cnt > 41:
+            raise ValueError(f"Expected 41 files, found {cnt:d}")
+        if cnt < 41:
+            fns = satpy.demo.download_fci_test_data()
+        self.filenames = fnmatch.filter(fns, "*-CHK-BODY-*")
+
+    def get_filenames(self):
+        """Get filenames of FCI test data as already available."""
+        p = satpy.demo.fci._get_fci_test_data_dir()
+        g = p.glob("UNCOMPRESSED/NOMINAL/*.nc")
+        return [os.fspath(fn) for fn in g]
+
+    def time_create_scene(self, chunk):
+        """Time to create a scene."""
+        self.create_scene(chunk)
+    time_create_scene.params = ["some", "all"]
```

Comment:
These parameters seem to appear in the results as `param1` and `param2` which is not very relevant (according to your paste there), is it possible to rename them if it is still the case?